android-patches/patches/V8/packages/rn-tester/android/app/build.gradle

```diff
@@ -0,0 +1,13 @@
+diff --git a/packages/rn-tester/android/app/build.gradle b/packages/rn-tester/android/app/build.gradle
```

Comment:
Note: Putting this in v8 as I will be adding additional patches for this file in a future PR, related to v8. If desired I can put it in a different directory.